src/api/data/cache/cache.go

```diff
@@ -0,0 +1,87 @@
+package cache
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/helm/monocular/src/api/data"
+	"github.com/helm/monocular/src/api/data/helpers"
+	"github.com/helm/monocular/src/api/mocks"
+	"github.com/helm/monocular/src/api/swagger/models"
+)
+
+var mockCharts = mocks.NewMockCharts()
+
+type cachedCharts struct {
+	knownRepos []map[string]string
+	allCharts  map[string][]*models.ChartVersion
+	rwm        *sync.RWMutex
+}
+
+// NewCachedCharts returns a new cachedCharts
+func NewCachedCharts(repos []map[string]string) data.Charts {
+	return &cachedCharts{
+		knownRepos: repos,
+		rwm:        new(sync.RWMutex),
```

Comment:
If rwm is not a mutex, you don't need this line since "the zero value for a RWMutex is an unlocked mutex."